### DISCUSSION TITLE
a service that can only be accessed from the internal network.

### DISCUSSION THREAD

**Discussion by sj6219** [rustdesk/rustdesk#1058] [General]
Currently, if you select the Enable Direct IP Access menu, it seems to make it easy to access not only from the outside but also from the inside. So it is always chosen and used. 

Besides this,  Direct IP Access Only menu is required. For security reasons, it is necessary to be inaccessible from the outside and accessible only from the inside.

**Comment by AlexPdx007** [comment]
So i understand that "Direct IP Access" uses port 21118 by default, but is the connection made through the Relay Server (own RustDesk server)? or is it P2P in this case ?
If i use it ONLY inside the network...and my Fortigate FortiGuard blocked this port ...i could not connect, and nobody could connect, from the outside (internet) ?
